### ISSUE TITLE
Add on does not install from within Orange

### ISSUE THREAD

**Issue by lawrencerowland** [biolab/orange3-network#97]
##### Network version
1.4.1

##### Orange version
3.18 on Mac 10.13.5

##### Expected behavior
Add-on installed when ticked and installed from menu

##### Actual behavior
Failure notice: "Command failed: python python -m pip install Orange3-Network exited with non zero status"

##### Additional info (worksheets, data, screenshots, ...)

ollecting Orange3-Network
  Using cached https://files.pythonhosted.org/packages/e9/86/78093ab58194619133289d1ffadc13a02ee3a61c479913b88fe8a7c2fa99/Orange3-Network-1.4.1.tar.gz
Requirement already satisfied: networkx>=2.0 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3-Network)
Requirement already satisfied: pyqtgraph>=0.9.10 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3-Network)
Requirement already satisfied: numpy>=1.8.0 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3-Network)
Requirement already satisfied: Orange3>=3.3.9 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3-Network)
Requirement already satisfied: decorator>=4.3.0 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from networkx>=2.0->Orange3-Network)
Requirement already satisfied: AnyQt>=0.0.8 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: bottleneck>=1.0.0 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: chardet>=3.0.2 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: docutils in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: fastTSNE>=0.2.12 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: joblib>=0.9.4 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: keyring in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: keyrings.alt in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: matplotlib>=2.0.0 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: pip>=9.0 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: python-louvain in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: requests in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: scikit-learn>=0.20.0 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: scipy>=0.16.1 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: serverfiles in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: setuptools>=36.3 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: xlrd>=0.9.2 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: numba>=0.38.1 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from fastTSNE>=0.2.12->Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: six in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from keyrings.alt->Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from matplotlib>=2.0.0->Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: python-dateutil>=2.1 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from matplotlib>=2.0.0->Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: kiwisolver>=1.0.1 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from matplotlib>=2.0.0->Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: cycler>=0.10 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from matplotlib>=2.0.0->Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: certifi>=2017.4.17 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from requests->Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: urllib3<1.25,>=1.21.1 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from requests->Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: idna<2.8,>=2.5 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from requests->Orange3>=3.3.9->Orange3-Network)
Requirement already satisfied: llvmlite>=0.25.0dev0 in /Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from numba>=0.38.1->fastTSNE>=0.2.12->Orange3>=3.3.9->Orange3-Network)
Building wheels for collected packages: Orange3-Network
  Running setup.py bdist_wheel for Orange3-Network: started
  Running setup.py bdist_wheel for Orange3-Network: finished with status 'error'
  Complete output from command /Applications/Orange3.app/Contents/MacOS/PythonApp -u -c "import setuptools, tokenize;__file__='/private/var/folders/19/r1rztl3d0yd03qc37zpnjpg40000gn/T/pip-build-5vgmwts6/Orange3-Network/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /var/folders/19/r1rztl3d0yd03qc37zpnjpg40000gn/T/tmp7upzq8h5pip-wheel- --python-tag cp36:
  running bdist_wheel
  running build
  running config_cc
  unifing config_cc, config, build_clib, build_ext, build commands --compiler options
  running config_fc
  unifing config_fc, config, build_clib, build_ext, build commands --fcompiler options
  running build_src
  build_src
  building extension "orangecontrib.network._fr_layout" sources
  build_src: building npy-pkg config files
  running build_py
  creating build
  creating build/lib.macosx-10.6-intel-3.6
  creating build/lib.macosx-10.6-intel-3.6/orangecontrib
  copying orangecontrib/__init__.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib
  creating build/lib.macosx-10.6-intel-3.6/orangecontrib/network
  copying orangecontrib/network/__init__.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network
  copying orangecontrib/network/community.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network
  copying orangecontrib/network/network.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network
  copying orangecontrib/network/readwrite.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network
  copying orangecontrib/network/setup.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network
  creating build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
  copying orangecontrib/network/widgets/__init__.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
  copying orangecontrib/network/widgets/graphview.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
  copying orangecontrib/network/widgets/OWNxAnalysis.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
  copying orangecontrib/network/widgets/OWNxClustering.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
  copying orangecontrib/network/widgets/OWNxExplorer.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
  copying orangecontrib/network/widgets/OWNxFile.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
  copying orangecontrib/network/widgets/OWNxFromDistances.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
  copying orangecontrib/network/widgets/OWNxGenerator.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
  copying orangecontrib/network/widgets/OWNxSave.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
  creating build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/airtraffic.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/airtraffic_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/dicty_publication.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/dicty_publication_edges.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/dicty_publication_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/lastfm.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/lastfm.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/lastfm_all.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/lastfm_tags.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/leu_by_genesets.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/leu_by_genesets_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/leu_by_pmid.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/leu_by_pmid_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/leu_huttenhower_a.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/leu_huttenhower_a_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/makeNetworkList.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/mips_c2_cp_leu.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/mips_c2_cp_leu_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  copying orangecontrib/network/networks/network_info.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
  creating build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/button_autoscale.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/button_rect.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/Category-Network.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/Dlg_clear.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/Dlg_HideSelection.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/Dlg_Mark2Sel.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/Dlg_Sel2Mark.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/Dlg_SelisMark.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/Dlg_UnselectedNodes.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/NetworkAnalysis.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/NetworkClustering.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/NetworkExplorer.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/NetworkFile.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/NetworkFromDistances.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  copying orangecontrib/network/widgets/icons/NetworkGenerator.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
  running build_ext
  customize UnixCCompiler
  customize UnixCCompiler using build_ext
  customize UnixCCompiler
  #### ['/usr/bin/clang', '-fno-strict-aliasing', '-Wsign-compare', '-fno-common', '-dynamic', '-DNDEBUG', '-g', '-fwrapv', '-O3', '-Wall', '-Wstrict-prototypes', '-arch', 'i386', '-arch', 'x86_64', '-g'] #######
  customize UnixCCompiler using build_ext
  building 'orangecontrib.network._fr_layout' extension
  compiling C++ sources
  C compiler: /usr/bin/clang++ -fno-strict-aliasing -Wsign-compare -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch i386 -arch x86_64 -g
  
  creating build/temp.macosx-10.6-intel-3.6/orangecontrib
  creating build/temp.macosx-10.6-intel-3.6/orangecontrib/network
  compile options: '-I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/numpy/core/include -I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/numpy/core/include -I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/include/python3.6m -c'
  clang++: orangecontrib/network/_fr_layout.cpp
  xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
  xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
  error: Command "/usr/bin/clang++ -fno-strict-aliasing -Wsign-compare -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch i386 -arch x86_64 -g -I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/numpy/core/include -I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/numpy/core/include -I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/include/python3.6m -c orangecontrib/network/_fr_layout.cpp -o build/temp.macosx-10.6-intel-3.6/orangecontrib/network/_fr_layout.o -MMD -MF build/temp.macosx-10.6-intel-3.6/orangecontrib/network/_fr_layout.o.d" failed with exit status 1
  
  ----------------------------------------
  Failed building wheel for Orange3-Network
  Running setup.py clean for Orange3-Network
Failed to build Orange3-Network
Installing collected packages: Orange3-Network
  Running setup.py install for Orange3-Network: started
    Running setup.py install for Orange3-Network: finished with status 'error'
    Complete output from command /Applications/Orange3.app/Contents/MacOS/PythonApp -u -c "import setuptools, tokenize;__file__='/private/var/folders/19/r1rztl3d0yd03qc37zpnjpg40000gn/T/pip-build-5vgmwts6/Orange3-Network/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /var/folders/19/r1rztl3d0yd03qc37zpnjpg40000gn/T/pip-lecl0oaz-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running config_cc
    unifing config_cc, config, build_clib, build_ext, build commands --compiler options
    running config_fc
    unifing config_fc, config, build_clib, build_ext, build commands --fcompiler options
    running build_src
    build_src
    building extension "orangecontrib.network._fr_layout" sources
    build_src: building npy-pkg config files
    running build_py
    creating build
    creating build/lib.macosx-10.6-intel-3.6
    creating build/lib.macosx-10.6-intel-3.6/orangecontrib
    copying orangecontrib/__init__.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib
    creating build/lib.macosx-10.6-intel-3.6/orangecontrib/network
    copying orangecontrib/network/__init__.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network
    copying orangecontrib/network/community.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network
    copying orangecontrib/network/network.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network
    copying orangecontrib/network/readwrite.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network
    copying orangecontrib/network/setup.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network
    creating build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
    copying orangecontrib/network/widgets/__init__.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
    copying orangecontrib/network/widgets/graphview.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
    copying orangecontrib/network/widgets/OWNxAnalysis.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
    copying orangecontrib/network/widgets/OWNxClustering.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
    copying orangecontrib/network/widgets/OWNxExplorer.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
    copying orangecontrib/network/widgets/OWNxFile.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
    copying orangecontrib/network/widgets/OWNxFromDistances.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
    copying orangecontrib/network/widgets/OWNxGenerator.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
    copying orangecontrib/network/widgets/OWNxSave.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets
    creating build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/airtraffic.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/airtraffic_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/dicty_publication.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/dicty_publication_edges.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/dicty_publication_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/lastfm.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/lastfm.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/lastfm_all.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/lastfm_tags.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/leu_by_genesets.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/leu_by_genesets_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/leu_by_pmid.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/leu_by_pmid_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/leu_huttenhower_a.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/leu_huttenhower_a_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/makeNetworkList.py -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/mips_c2_cp_leu.net -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/mips_c2_cp_leu_items.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    copying orangecontrib/network/networks/network_info.tab -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/networks
    creating build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/button_autoscale.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/button_rect.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/Category-Network.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/Dlg_clear.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/Dlg_HideSelection.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/Dlg_Mark2Sel.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/Dlg_Sel2Mark.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/Dlg_SelisMark.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/Dlg_UnselectedNodes.png -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/NetworkAnalysis.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/NetworkClustering.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/NetworkExplorer.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/NetworkFile.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/NetworkFromDistances.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    copying orangecontrib/network/widgets/icons/NetworkGenerator.svg -> build/lib.macosx-10.6-intel-3.6/orangecontrib/network/widgets/icons
    running build_ext
    customize UnixCCompiler
    customize UnixCCompiler using build_ext
    customize UnixCCompiler
    #### ['/usr/bin/clang', '-fno-strict-aliasing', '-Wsign-compare', '-fno-common', '-dynamic', '-DNDEBUG', '-g', '-fwrapv', '-O3', '-Wall', '-Wstrict-prototypes', '-arch', 'i386', '-arch', 'x86_64', '-g'] #######
    customize UnixCCompiler using build_ext
    building 'orangecontrib.network._fr_layout' extension
    compiling C++ sources
    C compiler: /usr/bin/clang++ -fno-strict-aliasing -Wsign-compare -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch i386 -arch x86_64 -g
    
    creating build/temp.macosx-10.6-intel-3.6/orangecontrib
    creating build/temp.macosx-10.6-intel-3.6/orangecontrib/network
    compile options: '-I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/numpy/core/include -I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/numpy/core/include -I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/include/python3.6m -c'
    clang++: orangecontrib/network/_fr_layout.cpp
    xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
    xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
    error: Command "/usr/bin/clang++ -fno-strict-aliasing -Wsign-compare -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch i386 -arch x86_64 -g -I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/numpy/core/include -I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/numpy/core/include -I/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/include/python3.6m -c orangecontrib/network/_fr_layout.cpp -o build/temp.macosx-10.6-intel-3.6/orangecontrib/network/_fr_layout.o -MMD -MF build/temp.macosx-10.6-intel-3.6/orangecontrib/network/_fr_layout.o.d" failed with exit status 1
    
    ----------------------------------------
Command "/Applications/Orange3.app/Contents/MacOS/PythonApp -u -c "import setuptools, tokenize;__file__='/private/var/folders/19/r1rztl3d0yd03qc37zpnjpg40000gn/T/pip-build-5vgmwts6/Orange3-Network/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /var/folders/19/r1rztl3d0yd03qc37zpnjpg40000gn/T/pip-lecl0oaz-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/var/folders/19/r1rztl3d0yd03qc37zpnjpg40000gn/T/pip-build-5vgmwts6/Orange3-Network/
You are using pip version 9.0.1, however version 18.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

**Comment by janezd** [comment]
@ajdapretnar, can you take this one? You know more about problems with add-on installation. It again seems that pip tries to compile instead of using a binary wheel.

**Comment by ajdapretnar** [comment]
@lawrencerowland Thank you for reporting this. Could you perhaps tell us how you installed Orange? I cannot replicate this on my machine, so I need to know the specifics. Did you have Orange installed before and you updated it recently? Did you have any Python version installed before installing Orange?
Thank you!

**Comment by ajdapretnar** [comment]
@lawrencerowland Thank you for the reply. I think there must be some old Orange file in your system that cause the issue. I would suggest uninstalling Orange completely, including some old files that might be left after the removal. Then please install 3.18 again. Please make sure you are using Miniconda installation.

**Comment by lawrencerowland** [comment]
hi Adja
Man thanks for the advice. You were right. I discovered some files in Application support folder for an older version. I also installed Xcode because I got a second error message (see footer).
This worked and I am very pleased with the network functionality that I am looking forward to exploring
Best wishes
Lawrence
-------
  xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
